tests/test_waterwheel.py

```diff
@@ -50,10 +67,19 @@ def test_common_uncommon_overlap(self):
         self.assertEqual(str(doc.ents[1].label_), 'RIVER')
         for ent in doc.ents:
             self.assertIsNotNone(ent._.wikilink)
-    
+
+        text = 'The Pacific has double the area and more than double the water volume of the Atlantic Ocean'
+        doc = self.nlp(text)
+        self.assertEqual(str(doc.ents[0]), 'Pacific')
+        self.assertEqual(str(doc.ents[0].label_), 'OCEAN')
+        self.assertEqual(str(doc.ents[1]), 'Atlantic Ocean')
+        self.assertEqual(str(doc.ents[1].label_), 'OCEAN')
+        for ent in doc.ents:
+            self.assertIsNotNone(ent._.wikilink)
```

Comment:
No need to test for Oceans and Provinces in this case unless there are any oceans or provinces with names that are stop words like "The" or "Is".